core/src/modules/configWriter.ts

```diff
@@ -1,12 +1,13 @@
 import { api } from "actionhero";
 import fs from "fs";
 import path from "path";
+import slugify from "slugify";
```

Comment:
Could we use slugify elsewhere in code config... perhaps when providing IDs via the CLI?

Reply:
Good idea! I'll add it to the generator.

Reply:
I looked at what the generator was doing and I ended up going back to a custom method. We had some behavior in the generator that slugify couldn't provide to us, so I ditched it and went with a shared and custom solution and added tests for the conditions we want to solve for.